<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login and Registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/destktopQT.jpg</normaloff>:/images/destktopQT.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>7.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
border-image:url(:/images/newQT.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{
border-image: url(:/images/newQT.jpg);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="topBlock" native="true">
      <property name="styleSheet">
       <string notr="true">#topBlock{
border-image:url(:/images/headerQT.jpg);

}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Login and Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Registration</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Create your ID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="createYourID"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Create your password:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="createYourPassword"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Your email address:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="emailAddress"/>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(85, 0, 255);

}

QPushButton:hover{
background-color: rgb(255, 0, 255);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(100, 0, 255);
}</string>
         </property>
         <property name="text">
          <string>REGISTER</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Please login</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Your ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yourID"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Your password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yourPassword"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(85, 0, 255);

}

QPushButton:hover{
background-color: rgb(255, 0, 255);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(100, 0, 255);
}</string>
         </property>
         <property name="text">
          <string>LOGIN</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
